controllers/syncer/csi_node.go

```diff
@@ -188,6 +188,20 @@ func (s *csiNodeSyncer) ensureContainersSpec() []corev1.Container {
 		},
 	}
 
+	livenessProbe.Ports = ensurePorts(corev1.ContainerPort{
+		Name:          constants.HealthPortName,
+		ContainerPort: int32(healthPort),
+	})
+
+	livenessProbeContainerHealthPort := intstr.FromInt(int(healthPort))
+	livenessProbe.LivenessProbe = ensureProbe(10, 3, 10, corev1.ProbeHandler{
+		HTTPGet: &corev1.HTTPGetAction{
+			Path:   "/healthz",
+			Port:   livenessProbeContainerHealthPort,
```

Comment:
use directly `intstr.FromInt(int(healthPort))`. No need of `livenessProbeContainerHealthPort`